backend/battles/models.py

```diff
@@ -6,6 +6,15 @@
 from users.models import User
 
 
+class BattleQuerySet(models.QuerySet):
```

Comment:
You wrote this code without using TDD. Where are the tests?

Reply:
Sorry, I thought the view test was already testing this part of the code.